### PR TITLE
Add concept notebook to track jdaviz pypi metrics

### PR DESCRIPTION
**Pull request by javerbukh**
### Description

This notebook allows us to us the `pypistats` api to track metrics of jdaviz downloads, including date of download, python version, and operating system.

Box link for jdaviz metrics: https://stsci.app.box.com/folder/157063321174

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [ ] Is a milestone set? Milestone is only currently required for PRs related to Imviz MVP.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: notebooks/concepts/pypi_metrics.ipynb [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2e1ab075",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "today = date.today()\n",
+    "write_to_filepath = f\"/Users/javerbukh/Documents/Miscellaneous/jdaviz_metrics_{today}.txt\"\n",
```
Comment (kecnry) [inline notebooks/concepts/pypi_metrics.ipynb:91]:
```suggestion
    "write_to_filepath = f\"./jdaviz_metrics_{today}.txt\"\n",
```

**Review by ibusko — APPROVED** [review]
That's a nice and useful admin tool!

**Comment by rosteen** [comment]
Any idea what the mirror/non-mirror stat is supposed to represent? Looks wonky (doesn't add up to 100%)

```
Last 180 days of downloads
| category        | percent | downloads |
|:----------------|--------:|----------:|
| with_mirrors    | 100.00% |     4,656 |
| without_mirrors |  44.82% |     2,087 |
| Total           |         |     4,656 |
```

File: notebooks/concepts/pypi_metrics.ipynb (hunk 1 above)
Comment (javerbukh) [reply notebooks/concepts/pypi_metrics.ipynb:91]: Literally kept reminding myself "take that out before you commit it"...

**Comment by javerbukh** [comment]
@rosteen This is what they have on that: https://pypistats.org/faqs#what-is-the-difference-between-without_mirrors-and-with_mirrors

**Review by pllim — APPROVED** [review]
If it works for you, good enough for me. Thanks!

**Comment by pllim** [comment]
Any other devs want to review? If not, this is good to merge.

**Pull request merged**